docs/get-started/totally-looks-like/index.md

```diff
@@ -0,0 +1,203 @@
+# Finetuning ResNet50 on Totally Looks Like Dataset
+
+In this tutorial, we will use Finetuner to finetune embeddings on [Totally Looks Like dataset](https://sites.google.com/view/totally-looks-like-dataset) (TLL) dataset.
+The dataset consists of 6016 pairs of images (12032 in total).
+
+```{tip}
+Totally-Looks-Like is a dataset and benchmark challenging machine-learned representations to reproduce human perception of image similarity. As shown below, each image patch in the left has a corresponded similar image patch in the right. 
+```
+
+```{figure} overview.png
+:width: 70%
+```
+
+
+The rationale for choosing TLL dataset is
+- it is a relatively small dataset and doesn't require training a large network from scratch. So we train only part of the ResNet-50 by freezing layers.
+- it consists of pairs of images that can be formed as positive pairs (same classes). A random image can be considered as a negative pair. We construct `triplet` and use the `TripletLoss`. 
+
+After fine-tuning, the distance between positive pairs is expected to be pulled closer, while the distance between positive and negative pairs is expected to be pushed away.
+
+## Environment & Data Preparation
+
+We will download `left.zip` and `right.zip`, as stated before,
+each of them consists of 6016 images which can be formed into pairs based on the same file name.
+
+
+```shell
+pip install gdown
+pip install finetuner
+pip install torchvision
+
+gdown https://drive.google.com/uc?id=1jvkbTr_giSP3Ru8OwGNCg6B4PvVbcO34
+gdown https://drive.google.com/uc?id=1EzBZUb_mh_Dp_FKD0P4XiYYSd0QBH5zW
+
+unzip left.zip
+unzip right.zip
+```
+
+Afterward, we load all images from unzipped `left` and `right` folders and turn them into sorted order as jina `DocumentArray`.
+80% of the dataset will be used to train the machine learning model.
+While 20% of the dataset will be used to evaluate the quality of embeddings on the search task.
+
+```python
+from docarray import DocumentArray
+
+left_da = DocumentArray.from_files('left/*.jpg')
+right_da = DocumentArray.from_files('right/*.jpg')
+# we use 80% for training machine learning model.
+left_da.sort(key=lambda x: x.uri)
+right_da.sort(key=lambda x: x.uri)
+
+ratio = 0.8
+train_size = int(ratio * len(left_da))
+
+train_da = left_da[:train_size] + right_da[:train_size]
+train_da = train_da.shuffle()
```

Comment:
I would remove this shuffle as it server no purpose. Real shuffling will be done by the class dataset